test/Microsoft.AspNet.Mvc.Testing.Test/ControllerTestHelperTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
@rynowak I looked into two controllers, AccountController and AlubmsApiController. I don't think we are interested in all the actions of the controllers. Many of them doesn't even need this feature to test. 
 And, I think the following actions should be good enough to show how users codes look like:
1. VerifyCode for using ModelState.isValid and ModelState.AddModelError
2. Register for Url.Action, RedirectToAction, ViewBag.Link View, AddErrors
3. ExternalLoginConfirmation for User.Identity.IsAuthenticated and  ModelState.IsValid

Let me know if you have any other actions to try out.